.travis.yml

```diff
@@ -5,11 +5,9 @@ language: ruby
 cache: bundler
 before_install: gem update bundler
 rvm:
-- 2.0
-- 2.1
-- 2.2
-- 2.3
 - 2.4
+- 2.5
+- 2.6
```

Comment:
We also would like to add support for 2.7. However, when I test this with 2.7, I get errors related to HTTP requests failing with vcr. Any ideas?